### PR TITLE
Update deps, fix tests

### LINKED ISSUE #34: An in-range update of next is breaking the build 🚨
## Version **6.1.2** of **next** was just published.

<table>
  <tr>
    <th align=left>
      Branch
    </th>
    <td>
      <a href="https://github.com/fastify/fastify-react/compare/master...fastify:greenkeeper%2Fnext-6.1.2">Build failing 🚨</a>
    </td>
  </tr>
  <tr>
    <th align=left>
      Dependency
    </th>
    <td>
      <a target=_blank href=https://github.com/zeit/next.js>next</a>
    </td>
  </tr>
  <tr>
    <th align=left>
      Current Version
    </td>
    <td>
      6.1.1
    </td>
  </tr>
  <tr>
    <th align=left>
      Type
    </td>
    <td>
      dependency
    </td>
  </tr>
</table>

This version is **covered** by your **current version range** and after updating it in your project **the build failed**.

next is a direct dependency of this project, and **it is very likely causing it to break**. If other packages depend on yours, this update is probably also breaking those in turn.

<details>
<summary>Status Details</summary>

- ❌ **continuous-integration/travis-ci/push:** The Travis CI build could not complete due to an error ([Details](https://travis-ci.org/fastify/fastify-react/builds/425681017?utm_source=github_status&utm_medium=notification)).
</details>

---

<details>
<summary>Release Notes</summary>
<strong>6.1.2</strong>

<p>This release is geared towards supporting <code>styled-components</code> v4.</p>
<h3>Patches</h3>
<ul>
<li>Update hoist-non-react-statics: <a class="issue-link js-issue-link" data-error-text="Failed to load issue title" data-id="357828032" data-permission-text="Issue title is private" data-url="https://github.com/zeit/next.js/issues/5116" href="https://urls.greenkeeper.io/zeit/next.js/pull/5116">#5116</a></li>
</ul>
<h3>Credits</h3>
<p>Huge thanks to <a class="user-mention" data-hovercard-user-id="570070" data-octo-click="hovercard-link-click" data-octo-dimensions="link_type:self" href="https://urls.greenkeeper.io/probablyup">@probablyup</a> for helping!</p>
</details>

<details>
<summary>Commits</summary>
<p>The new version differs by 3 commits.</p>
<ul>
<li><a href="https://urls.greenkeeper.io/zeit/next.js/commit/932e72b96523f649b19f2be1d6fb21436e4198cd"><code>932e72b</code></a> <code>6.1.2</code></li>
<li><a href="https://urls.greenkeeper.io/zeit/next.js/commit/33692a5678d2d0c06069580d340814497ebbf5ee"><code>33692a5</code></a> <code>update hoist-non-react-statics (#5116)</code></li>
<li><a href="https://urls.greenkeeper.io/zeit/next.js/commit/0fa642aacf524c6e60a6e8af212f4e13fdc20427"><code>0fa642a</code></a> <code>Update with-hashed-statics example (#5100)</code></li>
</ul>
<p>See the <a href="https://urls.greenkeeper.io/zeit/next.js/compare/0bcee3c23f170ae3caa3422c8253d3d1eeda8d74...932e72b96523f649b19f2be1d6fb21436e4198cd">full diff</a></p>
</details>

<details>
<summary>FAQ and help</summary>

There is a collection of [frequently asked questions](https://greenkeeper.io/faq.html). If those don’t help, you can always [ask the humans behind Greenkeeper](https://github.com/greenkeeperio/greenkeeper/issues/new).
</details>

---

Your [Greenkeeper](https://greenkeeper.io) Bot :palm_tree:

### LINKED ISSUE #39: fastify-react issue including a css file

### PR DESCRIPTION
**Pull request by cemremengu**
Closes #34, #40, #41 
Once published it will also close #39 too I think

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -6,6 +6,11 @@
   "engines": {
     "node": ">=8"
   },
+  "standard": {
+    "ignore": [
+      "*.jsx"
+    ]
+  },
```
Comment (lependu) [inline package.json]: Is this neccessary? For me standard handles `jsx` just fine. Do I miss something?

Comment (cemremengu) [reply package.json]: Yes, I couldnt figure out a better way than this unfortunately :/

**Comment by cemremengu** [comment]
@mcollina patch version is ok here right?

**Comment by mcollina** [comment]
A patch version is ok here.

**Pull request merged**